ChakraCore Samples/Hello World/Linux_OSX/Makefile

```diff
@@ -9,13 +9,16 @@ CC=g++
 LIBRARY_PATH=../BuildLinux/$(BUILD_TYPE)/lib
 LDIR=$(LIBRARY_PATH)/../pal/src/libChakra.Pal.a \
 	$(LIBRARY_PATH)/Common/Core/libChakra.Common.Core.a \
-	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a
+	$(LIBRARY_PATH)/Jsrt/libChakra.Jsrt.a \
+	/usr/local/opt/icu4c/lib/libicudata.a \
+	/usr/local/opt/icu4c/lib/libicuuc.a \
+	/usr/local/opt/icu4c/lib/libicui18n.a 
```

Comment:
`make` supports command line arguments. i.e. we may accept `${ICU_PATH}` and if it is empty, we may set the `/usr/local/opt/icu4c/` ?

So, if someone has `icu` is installed on another location, `make ICU_PATH=custom location here`